vendor/github.com/jesseduffield/gocui/gui.go

```diff
@@ -1311,6 +1311,10 @@ func (g *Gui) onKey(ev *GocuiEvent) error {
 	switch ev.Type {
 	case eventKey:
 
+		if g.IsPasting && ev.Key == KeyCtrlJ {
```

Comment:
This deserves a code comment explaining what we're doing. Even reading the commit message, it's not actually clear to me how KeyCtrlJ and KeyCtrlM relate to \n and \r.

Reply:
Added a comment in 0c0287c8cf17, does this help? It's still slightly mysterious to me, I tried to be honest about this in the comment.